src/Castle.Core.Tests/DynamicProxy.Tests/CustomModifiersTestCase.cs

```diff
@@ -11,7 +11,6 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-#if FEATURE_ASSEMBLYBUILDER_SAVE
```

Comment:
There doesn't appear to be a good reason (or I cannot remember it) why the whole test fixture file was guarded instead of just the bit where we try to save & verify the dynamic assembly.